src/lib/LocalStorage.ts

```diff
@@ -0,0 +1,31 @@
+import { Storage } from "./storage";
+
+class LocalStorage extends Storage {
+    set<todo>(key: string, value: todo): void {
```

Comment:
LocalStorage shouldn't care about todo, it should only care about key and value as long as both are strings

This should be made generic